renku/core/util/os.py

```diff
@@ -234,7 +241,7 @@ def hash_file(path: Union[Path, str], hash_type: str = "sha256") -> Optional[str
         return hash_file_descriptor(f, hash_type)
 
 
-def hash_str(content: str, hash_type: str = "sha256") -> str:
+def hash_string(content: str, hash_type: str = "md5") -> str:
```

Comment:
I removed this function since it's not used and we have it in the history.

`md5` is a bit faster than `sha256` and since this is used just for hashing (and not cryptography) and we might use it a lot, I've replaced the hash function.

Reply:
but wouldn't that cause issues with existing hashes?

Reply:
Right. I've used `md5` for new hash functions and this is the only old one that I've replaced. It's not used anywhere so no harm done.